app/views/shelters/_search_form.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_with url: path, method: method, local: true do |f| %>
+  
+  <%= f.label :search %>
+  <%= f.text_field :search %>
+  <%= f.submit "Search" %>
+<% end %>
```

Comment:
This partial is exactly the same as the other one. Since they are so similar, you can just have one and put it in a `app/views/shared/` folder. You would then be able to use them across any views.